tests/fixtures/api_controller.py

```diff
@@ -83,7 +83,6 @@ def __init__(
         self,
         db: DatabaseTransactionFixture,
         services_fixture: ServicesFixture,
-        setup_cm: bool,
```

Comment:
The `setup_cm` arg was only used in the scopedsession test, and its not needed there anymore due to the fixture changes. So we remove this options to simply the fixture a bit.